### ISSUE TITLE
Readme should mention Microsoft.CodeAnalysis.AnalyzerUtilities

### ISSUE THREAD

**Issue by jonas-johansson** [dotnet/roslyn-analyzers#3339]
The Readme should mention `Microsoft.CodeAnalysis.AnalyzerUtilities` as it's incredibly helpful.

I was looking for a way to calculate the cyclomatic complexity, and I found related code in `roslyn-analyzers`, but I couldn't see how it was publicly exposed. @mavasani enlightened me about `Microsoft.CodeAnalysis.AnalyzerUtilities` on [gitter](https://gitter.im/dotnet/roslyn?at=5e5d160b3ca0375cb396d4b0)).

Thanks for Roslyn and the related code, it's great!

**Comment by Evangelink** [comment]
@mavasani Is there a plan to share `AnalyzerUtilities`? If not then I am not sure what this readme mention would bring.

**Comment by mavasani** [comment]
Agreed. This is still an experimental package, so we don't need a mention in readme.
